packages/beacon-node/src/eth1/eth1DepositDataTracker.ts

```diff
@@ -175,16 +176,16 @@ export class Eth1DepositDataTracker {
           await sleep(sleepTimeMs, this.signal);
         }
       } catch (e) {
+        this.metrics?.eth1.depositTrackerUpdateErrors.inc(1);
+
         // From Infura: 429 Too Many Requests
         if (e instanceof HttpRpcError && e.status === 429) {
           this.logger.debug("Eth1 provider rate limited", {}, e);
           await sleep(RATE_LIMITED_WAIT_MS, this.signal);
-        } else if (!isErrorAborted(e)) {
-          this.logger.error("Error updating eth1 chain cache", {}, e as Error);
+          // only log error if state switched from online to some other state
+        } else if (!isErrorAborted(e) && this.eth1Provider.getState() !== oldState) {
```

Comment:
why are we comparing on oldstate since this is to wait `MIN_WAIT_ON_ERROR_MS` before retrying again if its some other error than aborted?